crates/host/src/wasmbus/mod.rs

```diff
@@ -340,9 +340,14 @@ impl Handler {
     #[instrument(skip(operation, request))]
     async fn call_provider_with_payload(
         &self,
+        target: Option<bus::host::TargetEntity>,
         operation: impl AsRef<str>,
         request: Vec<u8>,
     ) -> anyhow::Result<Result<Vec<u8>, String>> {
+        if target.is_some() {
+            // TODO: Implement
```

Comment:
nit: would be good to mark this and other TODOs as `TODO(#469)`, some GitHub plugins can link that up with the PR and later we can search for all the relevant TODOs